docs/internals-ja/git-workflow.md

```diff
@@ -55,15 +55,17 @@ phpunit をグローバルにインストールしていない場合は、代り
 
 取り組んでいるグループのものだけにテストを限定することが出来ます。
 例えば、バリデータと redis のためのテストだけを走らせるためには、`phpunit --group=validators,redis` とします。
-利用できるグループのリストを取得するためには、`phpunit --groups` を実行してください。
+利用できるグループのリストを取得するためには、`phpunit --list-groups` を実行してください。
 
 ### エクステンション
 
 エクステンションに取り組むためには、エクステンションのレポジトリをクローンする必要があります。
 私たちは、あなたに代ってそれをするコマンドを作っています。
 
-    php build/build dev/ext <extension-name>
-    
+```
+php build/build dev/ext <extension-name>
+```
```

Comment:
why? 4 space indentation is totally valid for code and more readable for one liners imo.